rpcs3/Emu/Cell/PPULLVMRecompilerCore.cpp

```diff
@@ -5207,11 +5207,11 @@ void Compiler::CreateBranch(llvm::Value * cmp_i1, llvm::Value * target_i32, bool
 			}
 
 			SetPc(target_i32);
-//			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
+			Function *fn = m_module->getFunction(fmt::format("function_0x%08X", target_address));
```

Comment:
@Zangetsu38 @vlj Added and uncommented it. There's probably a very good reason why it was commented - may it be due to a very specific issue in a very specific case or due to something else.

Once again:
It's like you removed the "elevator broken" sign from a broken elevator, rode up and down a couple times and now want people to use it.